bin/smart-proxy-abrt-send

```diff
@@ -10,21 +10,23 @@ $LOAD_PATH.unshift '/usr/share/foreman-proxy/modules'
 # can access our module's settings. Also used from smart-proxy code: global
 # settings, logging, foreman requests.
 require 'smart_proxy_main'
-
+require 'yaml'
 require 'smart_proxy_abrt'
 require 'smart_proxy_abrt/abrt_lib'
 
+$settings = YAML::load(File.open('/etc/foreman-proxy/settings.d/abrt.yml'))
```

Comment:
There's no need to use a ```$``` in front of a variable in ruby. This probably should be a constant though, so ```SETTINGS``` is more appropriate.